### ISSUE TITLE
No way for a new user to sign_up and use the app.

### ISSUE THREAD

**Issue by Vydruth** [doccano/doccano#1366]
The new UI is missing a `sign up` button for new users. Please See here: https://doccano.herokuapp.com/

It could be one of these things:
- Either the ALLOW_SIGNUP flag is not set.
- The new UI does not support signup. 
 
I think the issue is the latter option because I tried passing the ALLOW_SIGNUP flag locally and it did not work for me.

Signup Page: https://github.com/doccano/doccano/blob/master/backend/authentification/templates/signup.html#L5

@Hironsan @icoxfog417 Could you help verify the behavior to make sure it is actually a bug and not something I mistook?

**Comment by Vydruth** [comment]
Environment is not applicable for this issue.

**Comment by Vydruth** [comment]
This is really bad, it is a **SECURITY CONCERN** because I don't see any way to work around this, currently

- Users cannot SIgnup.
- Admin can create an account with a password, but the user cannot change their password on login password. **No Security!**

At least one of the features needs to be implemented for better security. Ideally, both features need to be implemented. Any suggestions on how one can get around this with the latest version of Doccano? @Hironsan @icoxfog417

**Comment by Hironsan** [comment]
Please use the admin page to create a user.

**Comment by Vydruth** [comment]
Hi Hiron, I hear you and I can do that, but it still is a bad security practice to create a username and password for a user and send it to them without allowing to change their password. Is there a way for the user to change their password in the new UI. I did not find one when I tried to change password. 

I am not sure if this issue should be closed before letting a user update their password or before giving an option for new users to sign up.
